materials/craft_items.lua

```diff
@@ -11,6 +13,7 @@ function nssm:register_eatcraftitems (name, descr, gnam)
         image = name..".png",
         on_use = minetest.item_eat(gnam),
         groups = { meat=1, eatable=1 },
+		_hunger_ng = {satiates = gnam},
```

Comment:
i dont know if this will need a conditional here! to detect if the mod are present and a dependency optional to hunger mod.. i will test and send a PR later